skimage/exposure/hdr.py

```diff
@@ -0,0 +1,308 @@
+import numpy as np
+
+
+def get_radiance(images, exposure, radiance_map):
+    """
+    Return the radiance for a series of images based upon a camera response
+    function.
+
```

Comment:
Add sentence to explain radiance

Reply:
Yup, you can add a Reference section for that. I also suggest renaming this function to `radiance`.

Reply:
I am hesitant to call i `radiance`, as radiance is a physical property, at this returns an image with the calculated radiance in every pixel. Maybe `radiance_image` is better.
I'll push a commit with a reference section.

Reply:
This method is now removed as it does not add anything and is full of errors